### PR TITLE
Changes at Development model

### PR DESCRIPTION
**Pull request by lciolecki**
* Add DemoFeatures category
* Some refactor data - cleaning

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -468,6 +468,7 @@ lazy val kafka = (project in engine("kafka")).
     name := "nussknacker-kafka",
     libraryDependencies ++= {
       Seq(
+        "javax.validation" % "validation-api" % javaxValidationApiV,
```
Comment (mproch) [inline build.sbt]: let's remove it for next PR :)

File: engine/flink/management/sample/src/main/scala/pl/touk/nussknacker/engine/management/sample/UnitTestsProcessConfigCreator.scala [hunk 2]
```diff
@@ -24,6 +24,7 @@ import pl.touk.nussknacker.engine.flink.util.sink.EmptySink
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.Random
 import UnitTestsProcessConfigCreator._
+import pl.touk.nussknacker.engine.management.sample.helper.DateProcessHelper
```
Comment (mproch) [inline engine/flink/management/sample/src/main/scala/pl/touk/nussknacker/engine/management/sample/UnitTestsProcessConfigCreator.scala]: do we need it?

File: engine/flink/management/sample/src/main/scala/pl/touk/nussknacker/engine/management/sample/dict/BusinessConfigDictionary.scala [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package pl.touk.nussknacker.engine.management.sample.dict
+
+import pl.touk.nussknacker.engine.api.dict.DictInstance
+import pl.touk.nussknacker.engine.api.dict.embedded.EmbeddedDictDefinition
+
+object BusinessConfigDictionary {
```
Comment (mproch) [inline engine/flink/management/sample/src/main/scala/pl/touk/nussknacker/engine/management/sample/dict/BusinessConfigDictionary.scala]: could we add some comment what feature it shows? At least for new components? After a while we forget about that :)

File: build.sbt (hunk 1 above)
Comment (lciolecki) [reply build.sbt]: I've added it in another PR: https://github.com/TouK/nussknacker/pull/809. There I will remove it.

File: engine/flink/management/sample/src/main/scala/pl/touk/nussknacker/engine/management/sample/UnitTestsProcessConfigCreator.scala (hunk 2 above)
Comment (lciolecki) [reply engine/flink/management/sample/src/main/scala/pl/touk/nussknacker/engine/management/sample/UnitTestsProcessConfigCreator.scala]: Yep.. 

```
  override def expressionConfig(config: Config) = {
    val globalProcessVariables = Map(
      "DATE" -> all(DateProcessHelper)
    )
    ExpressionConfig(globalProcessVariables, List.empty)
  }

File: ui/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/ManagementActor.scala [hunk 4]
```diff
@@ -254,18 +254,18 @@ class ManagementActor(managers: ProcessingTypeDataProvider[ProcessManager],
                                    latestVersion: ProcessVersionEntityData,
                                    savepointPath: Option[String],
                                    comment: Option[String])(implicit user: LoggedUser): Future[ProcessActionEntityData] = {
```
[the remark this answers is not in the document]
Comment (lciolecki) [reply ui/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/ManagementActor.scala]: I know and agree.. I've tried rollback old changes by using IDEA and history, result you can see now.. There is problem wich rollbak and formatting :|

File: engine/flink/management/sample/src/main/scala/pl/touk/nussknacker/engine/management/sample/UnitTestsProcessConfigCreator.scala (hunk 2 above)
Comment (mproch) [reply engine/flink/management/sample/src/main/scala/pl/touk/nussknacker/engine/management/sample/UnitTestsProcessConfigCreator.scala]: ah... I understand :)

**Pull request merged**